UPSTREAM-MERGE.sh

```diff
@@ -133,6 +128,16 @@ git commit -m "Merge upstream tag $version" -m "Operator SDK $version" -m "Merge
 # verify merge is correct
 git --no-pager log --oneline "$(git merge-base origin/"$rebase_branch" tags/"$version")"..tags/"$version"
 
+# update vendor directory
+go mod tidy
+go mod vendor
+# capture $? of go mod vendor, if it fails skip the add
```

Comment:
I'm still trying to understand the relevance of this comment line. Because neither we are capturing `$?` nor we are skipping the add when it fails.

Reply:
I can probably reword the comment.